telos-frontend/src/components/todo/DashboardTodo.jsx

```diff
@@ -38,62 +39,99 @@ const useStyles = makeStyles((theme) => ({
   datetextField: {
     marginLeft: theme.spacing(1),
     marginRight: theme.spacing(1),
-    width: 200,
+    width: '98%',
   },
 }));
 
-const listitems = [
-  {
-    name: 'OnGoing',
-    due: '18/03/2021',
-    onGoing: true,
-  },
-  {
-    name: 'OutDated',
-    due: '16/03/2021',
-    onGoing: false,
-  },
-];
-
 const outdated = {
   color: 'red',
 };
 
 const DashboardTodo = () => {
+  const listitems = [
+    {
+      name: 'OnGoing',
+      due: '2021-04-30',
+      onGoing: true,
+      completed: false,
+    },
+    {
+      name: 'OutDated',
+      due: '2021-01-30',
+      onGoing: false,
+      completed: true,
+    },
+    {
+      name: '一二三四五',
+      due: '2021-04-30',
+      onGoing: true,
+      completed: true,
+    },
+    {
+      name: '上山打老虎',
+      due: '2021-04-30',
+      onGoing: true,
+      completed: false,
+    },
+  ];
+
   const classes = useStyles();
-  const [checked, setChecked] = useState([0]);
   const [item, setItem] = useState('');
   const [dueDate, setDueDate] = useState('2021-01-01');
-  const [newItem, setNewItem] = useState(listitems);
+  const [itemList, setItemList] = useState(listitems);
+  const [newItem, setNewItem] = useState(itemList);
   const [anchorEl, setAnchorEl] = useState(null);
+  const [cancel, setCancel] = useState([0]);
+  const [reDate, setReDate] = useState('');
+  const [reItem, setReItem] = useState('');
+  const [add, setAdd] = useState(false);
 
-  const handleToggle = (value) => () => {
-    const currentIndex = checked.indexOf(value);
-    const newChecked = [...checked];
+  const reDateChange = (event) => {
+    setReDate(event.target.value);
+    setReItem({ name: reItem.name, due: event.target.value, onGoing: true, completed: false });
+  };
 
-    if (currentIndex === -1) {
-      newChecked.push(value);
-    } else {
-      newChecked.splice(currentIndex, 1);
+  const handleToggle = (value) => () => {
```

Comment:
if this only toggles one value, then would it be better to just include the index?